.nvmrc

```diff
@@ -0,0 +1 @@
+23
```

Comment:
do we recommend odd versions? 
at least in my day job, we try to stay on the prod versions. I suppose this IS a tooling author environment 😄

Reply:
> this IS a tooling author environment 😄

That's what I was thinking.

Reply:
may be use LTS for dev and recommend it for prod (cli usage of the tool)

Reply:
I think these would not be used in prod? @alexweej perhaps your use-case is different than what I'm imagining.